pom.xml

```diff
@@ -16,6 +16,7 @@
 
     <properties>
         <java.version>1.8</java.version>
+        <mapstruct.version>1.3.1.Final</mapstruct.version>
```

Comment:
The 1.0.0.Final is out of date. Compile wasn't recognizing the @Mapper annotation.